specification/maintenance/resource-manager/readme.python.md

```diff
@@ -0,0 +1,29 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml $(python)
+python-mode: create
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: azure.mgmt.maintenance
+  package-name: azure-mgmt-maintenance
+  package-version: 1.1.0
```

Comment:
Do we update this version?

Reply:
Hi @sotaneja , I think we can ignore this version.
The current Python codegen actually don't use package-version